### ISSUE TITLE
HTML Bars

### ISSUE THREAD

**Issue by Xenik007** [kingpin2k/HandlebarsHelper#6]
With the recent release of version 1.10 of EmberJs. Will you create an update for HTMLBars? I will try to look into it soon, but just tough i would ask first. I will try to create a pull request if i manage to get it working.

**Comment by kingpin2k** [comment]
Yeah, I am splitting it off though, I'm a little unsure of if I want to force the user to supply the version of the compiler and then avoid having to update the nuget package every time, or just update it every time.

**Comment by Xenik007** [comment]
you could default to the current version of the package but have an option to override the path to the javascript file?

**Comment by kingpin2k** [comment]
great idea, here's the initial commit, and it's working with the current
version, I'll toss in the override version later.  The only downside is if
the override version mucks up a variable name, then it'd require a code
change, I already have issue with their new precompiler, but I'll bring it
up with them.
https://www.nuget.org/packages/HTMLBarsHelper/
https://github.com/kingpin2k/HTMLBarsHelper

On Tue, Feb 10, 2015 at 7:28 AM, Xenik007 <notifications@github.com> wrote:

> you could default to the current version of the package but have an option
> to override the path to the javascript file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kingpin2k/HandlebarsHelper/issues/6#issuecomment-73707520>
> .
>



-- 
Daniel

Sent from my mind.

**Comment by Xenik007** [comment]
Great. Sounds good.
